### DISCUSSION TITLE
Phantom iOS browser sending GET instead of POST from submit buttons

### DISCUSSION THREAD

**Discussion by KeithWalton** [phantom/docs#102] [Help & Support] [answered]
### Summary

Starting at about the same time as the most recent Phantom iOS app update (4 days ago), my users have been reporting blank screens in my app when they click on buttons. By testing it and comparing to other browsers, I found that buttons of type submit are all sending GET requests instead of the expected POST when using the Phantom browser. You can see it in action here: 

https://pritv.io/explorer?collectionId=1 

Click on PriTV Toolkit. It will work in any browser except for Phantom (I've only tried IOS, not Android). The response code is 405 since the page doesn't support GET on that method.

### Example

https://pritv.io/explorer?collectionId=1

### Steps to Reproduce

Click on PriTV Toolkit. 

I'm using a .NET Core 7 web app.

This is what the button renders as:
<button type="submit" class="explorer-collection button btn" name="collectionId" value="2" onclick="waiting(this)" formaction="/Explorer/SelectCollection">PriTV Toolkit</button>

I also created a new project with just the starter page with a button. The same thing happens.

### Phantom Version

23.13.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hey @KeithWalton,

Thank you for writing in!

We're looking into this issue. I will follow up as soon as I have more info.

**Reply by KeithWalton** [reply]
Thanks. I found another way to reproduce it: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_form_method_post

If you enter data and click submit, the data won't show on the resulting page (compare to other browsers which show the data).

**Reply by adamdelphantom** [reply, marked as answer]
@KeithWalton We have a fix for this coming out in 23.15.0. Thank you again for reporting!

**Comment by KeithWalton** [comment]
Is there an estimated date of release?

**Reply by adamdelphantom** [reply]
It will likely be around 2-3 weeks.

**Comment by KeithWalton** [comment]
I see that 23.15.0 has been released, but the problem is still there. You
can reproduce it here:
http://solandia.io/postme

**Reply by adamdelphantom** [reply]
Thank you for this example app! We will investigate.

I noticed the pritv site appears to be rendering as expected now. Did you make updates to handle the post conversion issue?

**Reply by KeithWalton** [reply]
Any update on this? GETs really don't work well for us- when a user click the back button they get that annoying "are you sure you want to post again?" We can't bring on any new clients until this is resolved.

**Reply by adamdelphantom** [reply]
Hey @KeithWalton, I am sorry for the wait. We are investigating but I don't have any meaningful updates yet.

**Comment by KeithWalton** [comment]
Yes, we switched some of the posts on our site to gets, but there are still
some things broken.

**Comment by KeithWalton** [comment]
Any updates on this?

**Reply by adamdelphantom** [reply]
Hey @KeithWalton, nothing yet. Thanks for your patience! I'll let you know as soon as I have an update.
